Remotion/Web/Test.Shared/ExecutionEngine/ExceptionHandling/MissingPageFunction.cs

```diff
@@ -28,6 +28,6 @@ public MissingPageFunction ()
     {
     }
 
-    private WxePageStep Step1 = new WxePageStep ("~/ExecutionEngine/ExceptionHandling/MissingForm.aspx");
+    private WxeResourcePageStep Step1 = new WxeResourcePageStep (typeof (MissingPageFunction), "ExecutionEngine/ExceptionHandling/MissingForm.aspx");
```

Comment:
Type

Reply:
What type should I use if the `MissingForm.aspx` does not exist?

Reply:
Oh, okay. Right. so yeah, we keep it like that.